corona-warn-app/cwa-app-android
# Contact Journal time switching at midnight (EXPOSUREAPP-6092, EXPOSUREAPP-6451)

This PR addresses Exposureapp-6092 (Contact journal is switching too late).

**Problem-Description:**
Contact journal is switching /generating a new day one hour late (sometimes only many hours later as reports mention).
This creates a situation where user are not able to edit their diary for the new day e.g. right after midnight.

**How to test:**
- timetravel e.g. to 23:58
- switch to the CWA APP
- check if the Contact Journal creates a new Day at 0:00 (Journal Overview Screen)

->should happen automatically without reopening the diary now (with this fix)

**Screenshot:**
![diary_overview_better](https://user-images.githubusercontent.com/46747780/114856013-dbc39900-9de6-11eb-9605-a0d2d377d7b6.jpg)
